src/platforms/dialogflow/DialogFlowReply.ts

```diff
@@ -33,6 +33,12 @@ import { addToSSML, addToText, IVoxaReply } from "../../VoxaReply";
 import { DialogFlowEvent } from "./DialogFlowEvent";
 
 export interface IDialogFlowPayload {
+  facebook?: {
+    attachment: {
+      payload: any;
+      type: string;
+    };
+  };
```

Comment:
this now makes me think if google should also be an optional property

Reply:
I think it should be, but there are several parts that were breaking when I set it optional that I didn't want to spend more time on fixing, at least not now due to project pressure.